src/main/java/me/contaria/seedqueue/gui/wall/SeedQueueWallScreen.java

```diff
@@ -537,6 +538,7 @@ public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
         }
 
         if (SeedQueueKeyBindings.play.matchesKey(keyCode, scanCode)) {
+            this.showFinishedBenchmarkResults = false;
```

Comment:
Please remove this as toasts will be cleared by `SeedQueueWallScreen#remove` anyway

Reply:
> Please remove this as toasts will be cleared by `SeedQueueWallScreen#remove` anyway

done